kubetest/aksengine.go

```diff
@@ -340,14 +339,19 @@ func newAKSEngine() (*aksEngineDeployer, error) {
 		}
 	}
 
+	outputDir, err := ioutil.TempDir(os.Getenv("HOME"), "tmp")
```

Comment:
could we use os.TempDir() here?

Reply:
sure. changed

Reply:
actually, i'd prefer to keep this under a predictably named path so we can find the kubeconfig easily from other steps without access to this variable (I already have a need for this).